components/Layout/HomeHeader/HomeHeaderGenericTitle.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <p class="text-overline text-truncate mb-2 my-2">
```

Comment:
Also, I noticed you moved this line outside the main component. Why? It's repeated in all the three, it can live in the parent.